### PR TITLE
Suppress warnigs

### PR DESCRIPTION
**Pull request by supermarin**
Fixes in separate commit messages

### CODE REVIEW

File: Pod/Classes/LSTracer.m [hunk 1]
```diff
@@ -24,7 +24,7 @@
 @implementation LSTracer {
     NSString* m_accessToken;
     UInt64 m_runtimeGuid;
-    NSDictionary<NSString*, NSString*>* m_tracerJSON;
+    NSDictionary<NSString*, NSObject *>* m_tracerJSON;
```
Comment (bhs) [inline Pod/Classes/LSTracer.m:27]: why the NSObject here? (the values are intended to be `NSString*`s)

**Review by bhs — COMMENTED** [review]
(just the one question -- otherwise LGTM!)

File: Pod/Classes/LSTracer.m (hunk 1 above)
Comment (supermarin) [reply Pod/Classes/LSTracer.m:27]: @bensigelman if you look in the commit message body, we're passing  a dictionary as a value below :)

https://github.com/lightstep/lightstep-tracer-objc/pull/30/files#diff-dcd31561a7bbcafcbdeb70ddd1ba0b08R59

Comment (supermarin) [reply Pod/Classes/LSTracer.m:27]: That generated the warning in the first place

Comment (bhs) [reply Pod/Classes/LSTracer.m:27]: Ah :)   I misremembered which member this is. Generating JSON is such a bummer. Thanks.

**Review by bhs — APPROVED** [review]

**Pull request merged**